src/routes/github/subscription/github-subscription-delete.ts

```diff
@@ -38,7 +38,7 @@ export const GithubSubscriptionDelete = async (req: Request, res: Response): Pro
 			return;
 		}
 		try {
-			const subscription = await Subscription.getSingleInstallation(jiraHost, gitHubInstallationId);
```

Comment:
**NOTE**
Potential bug

Reply:
why?

Reply:
I mean before change, it is a bug. After change, it should be fine.